cmd/kubeadm/app/cmd/phases/join/preflight.go

```diff
@@ -98,7 +98,7 @@ func runPreflight(c workflow.RunData) error {
 	}
 
 	// Continue with more specific checks based on the init configuration
-	klog.V(1).Infoln("[preflight] Running configuration dependant checks")
+	klog.V(1).Infoln("[preflight] Running configuration dependent checks")
```

Comment:
technically, both are valid spellings of the word as a adjective:
https://dictionary.cambridge.org/dictionary/english/dependant
https://www.merriam-webster.com/words-at-play/spelling-variants-dependent-vs-dependant

yet, `ent` is more common in US English.

the change seems fine but should have been in a separate PR from the changelog change.

in any case:
/lgtm
/approve

i cannot approve the /CHANGELOG change.